### DISCUSSION TITLE
Clusters & bounding boxes

### DISCUSSION THREAD

**Discussion by feersantana5** [isl-org/Open3D#3722] [Q&A] [answered]
I would like to draw my point cloud (pcd) with the clusters (pcd_i) bounded with bounding boxes however I'm not able to use o3d.visualization.draw_geometries() in a proper way.

This is my code:

with o3d.utility.VerbosityContextManager(o3d.utility.VerbosityLevel.Debug) as cm:
        labels = np.array(pcd.cluster_dbscan(eps=1, min_points=1, print_progress=False))
    if len(labels) != 0:
        # Get biggest cluster label
        max_label = labels.max() + 1
          for i in range(max_label):
              # id is array of point names in main point cloud, owned by cluster with label i
              id = np.where(labels == i)[0]
              # Creating a new point cloud from cluster with label i
              pcd_i = pcd.select_by_index(id)
              # Bounding the cluster
              aabb = pcd_i.get_axis_aligned_bounding_box()
              aabb.color = (1,0,0)#aabb bounding box is red
              obb = pcd_i.get_oriented_bounding_box()
              obb.color = (0,1,0)#obbBounding box is green

o3d.visualization.draw_geometries(??????????????)

**Comment by hduonggithub** [comment, marked as answer]
o3d.visualization.draw_geometries([pcd, obb, aabb])
